news/service/Dockerfile

```diff
@@ -1,10 +1,11 @@
 FROM python:3.7.4 as builder
-WORKDIR /graphelier-news
-COPY requirements.txt .
-RUN pip install -r requirements.txt
+WORKDIR /news/service
 COPY . .
+RUN pip install -r requirements.txt
 
 FROM builder AS dev
+ENV FLASK_APP=server.py
+ENV FLASK_DEBUG=1
+ENV FLASK_ENV=development
 EXPOSE 8080
-ENTRYPOINT [ "python" ]
-CMD ["main.py"]
+CMD ["python", "server.py"]
```

Comment:
Could you add a prod entrypoint? The only way to run the news-service with your current setup is to use the dev docker-compose file